holopy/core/io/vis.py

```diff
@@ -72,6 +72,48 @@ def show(o, scaling='auto', vert_axis='x', horiz_axis='y',
     else:
         raise VisualizationNotImplemented(o)
 
+def save(o, names, scaling='auto', vert_axis='x', horiz_axis='y',
+                    depth_axis='z', colour_axis='illumination'):
+    """
+    Saves a hologram or reconstruction to (a) file(s).
+
+    Parameters
+    ----------
+    o : xarray.DataArray or ndarray
```

Comment:
I realize you are copying the syntax from `holopy.core.io.vis.show`, but can we change the variable name from `o` to something more descriptive? Perhaps `holograms` or `data`?

Reply:
Done!